src/app/ddGLWidgetView.cpp

```diff
@@ -2,11 +2,16 @@
 
 #include "ddOpenGL.h"
 
-#include <QGLWidget>
 #include <QVBoxLayout>
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+  #include <QOpenGLWidget>
+#else
+  #include <QGLWidget>
+  typedef QGLWidget QOpenGLWidget;
```

Comment:
If we require C++11, maybe write the less confusing `using QOpenGLWidget = QGLWidget`?